src/test/java/org/openmrs/contrib/qaframework/page/RegistrationSummaryPage.java

```diff
@@ -14,16 +14,48 @@
 
 public class RegistrationSummaryPage extends Page {
 
+	private static final By EDIT_DEMOGRAPHICS = By.cssSelector("#edit-patient-demographics a");
+	private static final By SHOW_CONTACT_INFO = By.id("patient-header-contactInfo");
+	private static final By EDIT_CONTACT_INFO = By.id("contact-info-inline-edit");
 	private static final By EDIT_RELATIONSHIP_LINK = By.id("relationships-info-edit-link");
+	private static final By PATIENT_GIVENNAME = By.cssSelector("#content div span.PersonName-givenName");
+	private static final By PATIENT_FAMILYNAME = By.cssSelector(".patient-header .demographics .name .PersonName-familyName");
+	private static final By TELEPHONE_NUMBER_TEXT = By.cssSelector("#coreapps-telephoneNumber");
 
-	public RegistrationSummaryPage(Page parent) {
-		super(parent);
```

Comment:
any reason for this change?

Reply:
@kdaud  one reviewer advised me that I should make my pages to extent page and therefore that section should reference page not parent.

Reply:
@kdaud  would be the best practice?

Reply:
is this a new page you are introducing? or its an existing one?

Reply:
> is this a new page you are introducing? or its an existing one?

It is an existing one, though I thought that if what I did was the best practice then it wouldn't matter whether the page is new or native. Just aligning stuff. :-)

Reply:
the child page is already extending the parent page through invoking the super constructor 

```js
public RegistrationSummaryPage(Page parent) {
	super(parent);
}
```
`parent/page` is just a variable name.

Reply:
> `parent/page` is just a variable name.

@kdaud , Would it be terrible if I called my variable page instead of parent?

Reply:
as mentioned earlier parent/page is just name. Variable names should tell what the variable is all about, so I would go with `parent.`

Reply:
Hi @kdaud , I have attended to the reviews.